embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -324,16 +325,17 @@ private void startImpl(@NonNull Context context,
         sessionOrchestrator = sessionModule.getSessionOrchestrator();
         sessionPropertiesService = sessionModule.getSessionPropertiesService();
 
+        Systrace.startSynchronous("send-cached-sessions");
         // Send any sessions that were cached and not yet sent.
         deliveryModule.getDeliveryService().sendCachedSessions(
             nativeModule.getNdkService(),
             essentialServiceModule.getSessionIdTracker()
         );
+        Systrace.endSynchronous();
 
         final CrashModule crashModule = moduleInitBootstrapper.getCrashModule();
         loadCrashVerifier(crashModule, moduleInitBootstrapper.getWorkerThreadModule());
 
-        Systrace.startSynchronous("internal-interface-init");
```

Comment:
Is there a reason these are removed?

Reply:
Yeah this is actually pretty fast now with a change I did earlier to not always initialize the hosted SDK's respective internal interfaces, so I got rid of it.